modules/ROOT/pages/getting-started/connect-instance.adoc

```diff
@@ -37,4 +35,28 @@ The default is `neo4j+s//`.
 
 |Password
 |You are given the password when you initially create the instance
-|===
\ No newline at end of file
+|===
+
+== Connection method
+
+To connect to an instance, you can choose from the following connection methods: Neo4j or Query API.
+
+*Neo4j*: The default is The default is Neo4j Bolt `neo4j+s//`.
+Connect through the Neo4j Bolt protocol using a WebSocket connection to the Neo4j port. 
+Use for best performance and stability.
+
+*Query API*: Connect to Neo4j through the Query API over HTTPS on port 443 (a standard, widely open port). 
+Use for debugging or when Bolt is unavailable due to network configuration.
+
+The connection method protocol is used for the communication between the Neo4j database server and the client application or tool.
+
+For more information about connection schemes, see link:https://neo4j.com/docs/operations-manual/current/configuration/connectors/[Operations Manual -> Configure network connectors] and link:https://neo4j.com/docs/bolt/current/bolt/[Bolt Protocol].
+
+== Errors 
+
+In an enterprise environment with restrictive networking, connection errors are sometimes caused because non-standard ports used by the Bolt protocol are blocked by firewalls.
+
+Neo4j tools have traditionally connected over Bolt, a binary protocol that offers optimal performance and stability, that requires an open WebSocket connection to the Neo4j port.
+
```

Comment:
```suggestion
```
Is this true? I think `neo4j` has been around for quite some time and as mentioned above, it's built on bolt but it is not the same, so we should probably say that.